user-interface/src/case-detail/CaseDetailScreen.tsx

```diff
@@ -240,6 +242,17 @@ export const CaseDetail = (props: CaseDetailProps) => {
     }
     setDocumentNumber(newDocumentNumber);
   }
+  function clearFilters() {
```

Comment:
Nit-pick, but could we add a newline above?
```suggestion

  function clearFilters() {
```

Reply:
Yes I'll do that now.